### PR TITLE
Style learning objectives for astronomy

### PR DESCRIPTION
**Pull request by kswanson33**
#3025

image

### CODE REVIEW

File: styles/designs/cardboard/parts/_learning-shapes.scss [hunk 1]
```diff
@@ -13,4 +13,20 @@
       ),
     ))
   ),
+));
+
+@include create_shape('cardboard:::LearningObjectivesCustomShape', (
```
Comment (ajluk) [inline styles/designs/cardboard/parts/_learning-shapes.scss:18]: @kswanson33 Could you add a comment that this shape is used only in Astronomy?

**Review by ajluk — COMMENTED** [review]
Everything looks good. I have just left one comment.

File: styles/books/astronomy/book.scss [hunk 2]
```diff
@@ -27,7 +27,18 @@ $bandColor: #772949;
 ));
 
 
+@include add_settings((
+  LearningObjectivesAstro: (
```
Comment (ajluk) [inline styles/books/astronomy/book.scss:31]: Here maybe the better title would be `LearningObjectivesCustom`. What do you think?

Comment (kswanson33) [reply styles/books/astronomy/book.scss:31]: Agreed -- for consistency's sake, either all settings should include 'Astronomy' in the name, or none of them should. I'm with you that none seems like the better choice.

**Review by ajluk — APPROVED** [review]

**Comment by kswanson33** [comment]
Git tangle alert! This is already merged into a.a., and all that's left is changes from master that weren't in z.z (and somehow made it into this branch before a.a.). Sorry!

**Pull request merged**